src/Compilers/CSharp/Portable/Binder/Semantics/BestTypeInferrer.cs

```diff
@@ -11,7 +11,12 @@ internal static class BestTypeInferrer
     {
         public static TypeSymbolWithAnnotations InferBestType(ImmutableArray<TypeSymbolWithAnnotations> types, Conversions conversions, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
```

Comment:
>InferBestType [](start = 48, length = 13)

Did you do a diff with master? It sounds like only the first two methods are new and the rest is back to original.